### PR TITLE
Castro handle downsize param

### PR DESCRIPTION
**Pull request by bernii**
@devlina please review
cc @saucelabs/autdev

### CODE REVIEW

File: castro/__init__.py [hunk 1]
```diff
@@ -114,19 +129,32 @@ def encode(self):
...
                    " -me_method dia -subq 0 -me_range 16 -g %s -keyint_min 25"
                    " -sc_threshold 0 -i_qfactor 0.71 -b_strategy 0 -qcomp 0.6"
                    " -qmin 10 -qmax 51 -qdiff 4 -bf 0 -refs 1 -directpred 1 -trellis 0"
                    " -flags2 -bpyramid-mixed_refs-wpred-dct8x8+fastpskip-mbtree -wpredp"
-                   " 0 -aq_mode 0 -crf 30 %s")
+                   " 0 -aq_mode 0 -crf 30 %s%s")
 
         os.system(cmd % (self.filepath,
                          self.framerate * self.seconds_per_keyframe,
+                         newsize,
```
Comment (devlina) [inline castro/__init__.py:161]: Not reducing frame rate anymore?

**Comment by devlina** [comment]
Why changed file permission on deploy.sh?

File: castro/__init__.py (hunk 1 above)
Comment (bernii) [inline castro/__init__.py:161]: I'm not 100% sure what are allowed params here, will try resolution reduction first :)

**Comment by bernii** [comment]
accidental when testing things, reverting :-)

**Pull request merged**